tests/models/resource_test.py

```diff
@@ -63,6 +67,9 @@ def setUpClass(cls):
         cls.user = User.objects.create_user("test", "test@archesproject.org", "password")
         cls.user.groups.add(Group.objects.get(name="Guest"))
 
+        graph = Graph.objects.get(pk=cls.search_model_graphid)
```

Comment:
it would appear that this graph should already be published based on [the fixture](https://github.com/archesproject/arches/blob/8e691a04fefec858f987358a0b2809fd37738f6c/tests/fixtures/resource_graphs/Resource%20Test%20Model.json#L510-L516).  Can we determine why it is not?